tools/scripts/l18n/pre_translation.sh

```diff
@@ -1,10 +1,9 @@
-#!/bin/bash
+#!/bin/sh
 
 # Extract Strings from API & UI
-make docker-compose-sources
-docker-compose -f tools/docker-compose/_sources/docker-compose.yml run awx_1 make awx-link migrate po messages
-
-# Move extracted Strings to Translation Directory
-mv awx/locale/en-us/LC_MESSAGES/django.po translations/
-mv awx/ui/src/locales/en/messages.po translations/
+make docker-compose-sources && \
+  docker-compose -f "${SOURCES:-_sources}"/docker-compose.yml run awx_1 make awx-link migrate po messages && \
```

Comment:
Just noticed this. Leaving a comment for myself to fix it: `_sources` should be `tools/docker-compose/_sources` or wherever it ends up when run normally.